app/src/main/java/com/orange/ods/app/ui/MainScreen.kt

```diff
@@ -118,11 +115,9 @@ fun MainScreen(themeConfigurations: Set<OdsThemeConfigurationContract>, mainView
                 modifier = Modifier
             }
 
-            val context = LocalContext.current
-            val aboutConfiguration = aboutConfiguration()
-            val aboutModule = odsAbout(context = context, configuration = aboutConfiguration)
-            val aboutCustomizationViewModel = viewModel<AboutCustomizationViewModel>(context as ViewModelStoreOwner)
-            val customAboutConfiguration = aboutCustomizationViewModel.aboutConfiguration()
+            val aboutDefaultConfiguration = aboutConfiguration()
```

Comment:
As discussed together, I suggest to rename some things.
`aboutDefaultConfiguration` is the app about configuration. Can be rename into `appAboutConfiguration`?
In this case should we rename the method `aboutConfiguration` into `appAboutConfiguration()` to be more clear?

Actually we have the About of the app and the demo of the about module. I think we should be able to distinct this two elements by reading the code. So maybe we have to change `navigateToAboutModule` into `navigateToAboutDemo` and `aboutCustomizationConfiguration` into `aboutDemoConfiguration`. What do you think?